install.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+declare -A deps
+deps["lateralgm.jar"]="IsmAvatar/LateralGM"
+deps["plugins/enigma.jar"]="enigma-dev/lgmplugin"
+
+get_latest () {
+  local release=$(curl --silent "https://api.github.com/repos/$1/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/')
+  echo "$release"
+}
+
+grab_latest() {
+  # 1 = repo path, 2 = version, 3 = local file
+  local fileName=$(basename $3)
+  echo -e "\e[32mAttempting to download $fileName to ${PWD}/$3 from https://github.com/$1/releases/download/$2/$fileName \e[0m"
+  curl -L -o "$3" "https://github.com/$1/releases/download/$2/$fileName"
+}
+
+mkdir -p "plugins/shared"
+
+#This is some ancient jna version we depend on 
+if [ -f "plugins/shared/jna.jar" ]; then
+  echo -e "plugins/shared/jna.jar \e[32mAlready up to date skipping...\e[0m"
+else
+  echo -e "\e[32mAttempting to download jna.jar to ${PWD}/plugins/shared/jna.jar from https://dl.dropboxusercontent.com/s/8n2pgy9qhsnc972/jna.jar \e[0m"
+  curl -L -o "plugins/shared/jna.jar" "https://dl.dropboxusercontent.com/s/8n2pgy9qhsnc972/jna.jar"
```

Comment:
Haven't we learned our lesson about Dropbox, yet...? Is this at least Robert's Dropbox (since he's the only one who's ever going to update LGM)? If not, I'll host the ball on ENIGMA's server. Or, you know, someone could upload it to GitHub.